client/view/Form/Form.css

```diff
@@ -11,13 +11,15 @@
 .Form__queryTextArea {
   box-sizing: border-box;
   width: 100%;
+  min-height: 150px;
```

Comment:
I wanted to take the value from the design and specify it in em. But so far I've done px, because now we have standard browser font-size and line-height